### ISSUE TITLE
[BUG] (liveclientdata) TFT currentGold does not represent correct data. Incrementally increases without gold gain/decrease

### ISSUE THREAD

**Issue by Its-Haze** [RiotGames/developer-relations#865]
**Bug Description**  
The endpoint ``https://127.0.0.1:2999/liveclientdata/activeplayer`` shows the currentGold value.
However that value does not correctly correspond to the actual gold my activeplayer has in game.

**Problem Description**  
The value of ``currentGold`` does not represent my actual current gold in game.

**Expected Result**  
Expecting ``curentGold`` value to represent my in game gold amount in ``Integer`` (whole numbers). since that is what's represented in the game.

**Actual Result**  
currentGold shows crazy high numbers with decimals, which does not make much sense.

Ingame: 
![image](https://github.com/RiotGames/developer-relations/assets/90850836/a15af6a5-e769-42df-b9e9-1224565d5bd8)

The API: "``https://127.0.0.1:2999/liveclientdata/activeplayer``"
![image](https://github.com/RiotGames/developer-relations/assets/90850836/3284c6cf-44c9-4d0b-99bb-f73881cf187f)

**Developer Impact**  
Developing stuff around the riot API is very difficult when values don't match what the expectations are.

* Frequency: Every API call.
* Severity: Annoying, and not usable field.
* User emails: I get feature requests to know the current ingame gold in my application, but that's simply not possible at this point in time.

**Repro Steps**  

1. Start a TFT game (Any game mode)
2. Go to your browser and put ``https://127.0.0.1:2999/liveclientdata/activeplayer`` in the URL field.
3. find the field "currentGold"
4. Observe it's value while you are playing the game.. refresh it once in a while (even if your ingame gold does not change, notice the currentGold value from the API)

**Comment by bangingheads** [comment]
Keep in mind this is a League of Legends API and not listed as a TFT API.

**Comment by Its-Haze** [comment]
@bangingheads  Sure, but if the game mode is "TFT"

And the field "level" corresponds to the actual level i have in TFT (which is great!).. Then I would also assume/expect the "currentGold" value to correspond to the actual gold I have too.
Right?? Or am i missing something here?

**Comment by bangingheads** [comment]
Riot has never stated that the In Game API is supported for TFT nor does it say so in documentation so I was just stating that it is unlikely they will fix bugs for an unsupported use case.

Things that show correctly are a consequence of them sharing the same values as League of Legends.

**Comment by thomasmarton** [comment]
Little fun fact for you: The currentGold value you see is actually your "summoner's rift gold", you can even see it count up passively just like it would for league.

TFT is still a league game mode after all and it made more sense for them to create a separate system for your TFT gold. But since some things in League are baked in deep (like the fact that to this day, you are basically playing Kaisa when playing TFT) into the game, the league gold system is still there, counting happily in the background.

Source: 2 TFT engineers that are no longer working on TFT :/

TLDR: The value is accurate, just not in the way you want it to be

**Comment by Its-Haze** [comment]
Understood. thanks
